### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by supmyhava** [abyansetya/moneydogs#1]
Trying to complete task: Telegram Premium
Failed to verify task 3: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/3/verify

Trying to complete task: Invite 5 friends to MONEY DOGS
Failed to verify task 4: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/4/verify

Trying to complete task: Invite 10 friends to MONEY DOGS
Failed to verify task 5: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/5/verify
root@od-big:~/moneydogs# nano query.txt
root@od-big:~/moneydogs# python3 moneydogs.py

Trying to complete task: Connect Wallet
Failed to verify task 2: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/2/verify

Trying to complete task: Telegram Premium
Failed to verify task 3: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/3/verify

Trying to complete task: Invite 5 friends to MONEY DOGS
Failed to verify task 4: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/4/verify

Trying to complete task: Invite 10 friends to MONEY DOGS
Failed to verify task 5: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/5/verify

Trying to complete task: Send your invite URL to Binance.com
Failed to verify task 28: Expecting value: line 1 column 1 (char 0)

Trying to complete task: Send your invite URL to okx.com
Failed to verify task 30: Expecting value: line 1 column 1 (char 0)

Trying to complete task: Join Mnemonics
Failed to verify task 32: 400 Client Error:  for url: https://api.moneydogs-ton.com/tasks/32/verify

Trying to complete task: Follow MONEY DOGS on X
Failed to verify task 33: Expecting value: line 1 column 1 (char 0)

Trying to complete task: Celebrate Gift for $HMSTR & $CATI users!
Failed to verify task 51: Expecting value: line 1 column 1 (char 0)

**Comment by abyansetya** [comment]
yeah man idk if this script still working or not. But the last time i use it, it will print error like that but if you check in the mini app, the task is already completed
